ci-operator/jobs/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main-periodics.yaml

```diff
@@ -13,13 +13,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-shiftstack-shiftstack-ci-main-cleanup-kuryr
-  reporter_config:
```

Comment:
I'm not sure why we're removing the reporting here but we don't want that.

Reply:
I'm not sure either. Must be some weirdness in updating jobs after rebase. Re-added all  the reports back, and add it to the migration job.